src/views/preview/project-view.jsx

```diff
@@ -243,13 +243,23 @@ class Preview extends React.Component {
                                 }
                             });
                         }
-                        const sprites = projectData[0].targets.length - 1; // don't count stage
-                        const scripts = projectData[0].targets
-                            .map(target =>
-                                Object.values(target.blocks)
-                                    .filter(block => block.topLevel).length
-                            )
-                            .reduce((accumulator, currentVal) => accumulator + currentVal, 0);
+                        let sprites = 0;
```

Comment:
can we rename these to something like `numSprites` and `numScripts`, even just in this block? It would help make their purpose clearer, even if we leave their names under `modInfo` the same.

Reply:
renamed everything to `...Count`